cclib/attribute_parsers/scftargets.py

```diff
@@ -61,6 +60,30 @@ def gaussian(file_handler, ccdata) -> Optional[dict]:
             return {scftargets.__name__: constructed_scftargets}
         return None
 
+    @staticmethod
+    def psi4(file_handler, ccdata) -> Optional[dict]:
+        line = file_handler.last_line
+        if (line.strip() == "==> Algorithm <==") and (getattr(ccdata, "finite_difference") is None):
+            file_handler.skip_lines(["blank"], virtual=True)
+            line = file_handler.virtual_next()
+            while line.strip():
+                if "Energy threshold" in line:
+                    etarget = float(line.split()[-1])
+                if "Density threshold" in line:
+                    dtarget = float(line.split()[-1])
+                line = file_handler.virtual_next()
+
+            if getattr(ccdata, "scftargets") is None:
+                this_scftargets = []
+            else:
+                this_scftargers = ccdata.scftargets
```

Comment:
```suggestion
                this_scftargets = ccdata.scftargets
```
Does this bug exist in the original code?

Reply:
haha no i doubt it, probably just trying to go too quickly on my end.

Reply:
This is fixed now!